.travis.yml

```diff
@@ -28,10 +30,11 @@ install:
   - ./travis-tool.sh r_binary_install servr
```

Comment:
Good catch. I think it's safer to assume people will have the CRAN version so I deleted the install_github line